docs/README.md

```diff
@@ -1,30 +1,33 @@
 # Cloud Cluster Management with vfxt.py
 
-The vfxt.py script is a command-line tool for creating and managing Avere clusters in cloud-based virtual computing environments.
+The vfxt.py script is a command-line tool for creating and managing Avere clusters in cloud-based virtual computing environments. 
 
-This script has complete capabilities for creating and managing Avere vFXT clusters, including:
+This script has complete capabilities for creating and managing Avere vFXT clusters, including: 
 
 * Create a new Avere vFXT cluster - including creating the vFXT nodes that make up the cluster and configuring cloud storage as a backend core filer
-* Destroy existing clusters (including the vFXT nodes)
+* Destroy existing clusters (including the vFXT nodes) 
 * Create and add new nodes to a cluster
 * Basic cluster configuration tasks
 
-For ongoing cluster administration, use the Avere Control Panel. Read the Avere cluster [Configuration Guide](<https://azure.github.io/Avere/#operations>) for more details.
```

Comment:
library.averesystems.com only redirects to azure.github.io. There's no guarantee that the redirect will continue to function so it's better to reference the github page directly.